package.json

```diff
@@ -24,5 +24,8 @@
     "eslint": "^8.28.0",
     "eslint-config-turbo": "latest",
     "turbo": "^1.9.4"
+  },
+  "dependencies": {
+    "@headlessui/react": "^1.7.14"
   }
```

Comment:
Why are we adding it here? Shouldn't it be in the app? (I think this also applies to `@svgr/webpack` now that I see it). I think this should only contain libs that are shared cross packages

Reply:
You are right, should be under the app.

fixed: [ecc9bb8](https://github.com/SwaprHQ/stackly-ui/pull/15/commits/ecc9bb844de25655e8012923db90643789a252cd)